nautobot_firewall_models/migrations/0021_alter_addressobject_status_and_more.py

```diff
@@ -0,0 +1,166 @@
+# Generated by Django 4.2.14 on 2024-08-06 00:36
+
+from django.db import migrations
+import django.db.models.deletion
+import nautobot.extras.models.statuses
+import nautobot_firewall_models.utils
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("extras", "0113_saved_views"),
```

Comment:
Is this migration backwards compatible with Django 3 or does this make Nautobot 2.3 the new required minimum version?

Reply:
It should be good now that I removed this dependency on extras 0113. It looks like the behavior change is only in the generation of migration files and in my testing these migrations are backward compatible.